scripts/mkcloud

```diff
@@ -1099,6 +1099,8 @@ Optional
         a fail will not be considered for now.
     want_efi=1 (default=0)
         Deploy nodes with EFI. Only available from Cloud7 and later.
+    want_keystone_token_type=fernet (default=uuid)
```

Comment:
I left default value empty, and added comment that default will be applied from Keystone barclamp